python/example_robot_data/robots_loader.py

```diff
@@ -50,19 +50,31 @@ class RobotLoader(object):
     path = ''
     urdf_filename = ''
     srdf_filename = ''
+    sdf_filename = ''
     urdf_subpath = 'robots'
     srdf_subpath = 'srdf'
+    sdf_subpath = ''
     ref_posture = 'half_sitting'
     has_rotor_parameters = False
     free_flyer = False
     verbose = False
 
     def __init__(self):
-        urdf_path = join(self.path, self.urdf_subpath, self.urdf_filename)
-        self.model_path = getModelPath(urdf_path, self.verbose)
-        self.urdf_path = join(self.model_path, urdf_path)
-        self.robot = RobotWrapper.BuildFromURDF(self.urdf_path, [join(self.model_path, '../..')],
-                                                pin.JointModelFreeFlyer() if self.free_flyer else None)
+        if self.urdf_filename:
+            if self.sdf_filename:
+                raise AttributeError("Please choose between URDF *or* SDF")
+            df_path = join(self.path, self.urdf_subpath, self.urdf_filename)
+            builder = RobotWrapper.BuildFromURDF
+        else:
+            df_path = join(self.path, self.sdf_subpath, self.sdf_filename)
+            try:
+                builder = RobotWrapper.BuildFromSDF
+            except AttributeError:
+                raise ImportError("Building SDF models require pinocchio >= 2.9.2")
```

Comment:
```suggestion
                raise ImportError("Building SDF models require pinocchio >= 2.9.1")
```

Reply:
I would suggest using Pinocchio >= 3.0.0. 
This is much more simple, as 2.9.x won't be released on Conda, APT, etc.

Reply:
Ok for 3. It's a little lie, but it won't hurt users who have 2.9 :)